src/services

```diff
@@ -733,6 +733,23 @@ p({
   "serve": true,
   "serviceClass": "foam.nanos.session.SimpleSessionService"
 })
+
+p({
+  "class":"foam.nanos.boot.NSpec",
+  "name":"notificationSettingDAO",
+  "serviceScript":"""
+    return new foam.dao.EasyDAO.Builder(x)
+      .setSeqNo(true)
+      .setPm(true)
+      .setJournalType(foam.dao.JournalType.SINGLE_JOURNAL)
+      .setJournalName("notificationSettings")
+      .setOf(foam.nanos.notification.NotificationSetting.getOwnClassInfo())
+      .setAuthorize(false)
```

Comment:
Is this safe? Probably not. Any user with access to this service will be able to delete/modify the settings of other users.